pkg/remote/remote.go

```diff
@@ -91,7 +91,7 @@ ARG {{ .GitCommitArgName }}
 ARG {{ .GitBranchArgName }}
 ARG {{ .InvalidateCacheArgName }}
 
-RUN echo "${{ .InvalidateCacheArgName }}" > /var/lib/.oktetocachekey
+RUN mkdir -p /var/lib && echo "${{ .InvalidateCacheArgName }}" > /var/lib/.oktetocachekey
```

Comment:
why do we need to write `. InvalidateCacheArgName` to `/var/lib/.oktetocachekey`?

Reply:
Why is it relevant to if the resulting image is scratch or not? we never push it
I think setting the `target` of the build request if there are no artifacts would be a better approach than writing to `/var/lib/.oktetocachekey`